packages/devtools_app/lib/src/config_specific/flutter/import_export/import_export.dart

```diff
@@ -14,10 +13,8 @@ const nonDevToolsFileMessage = 'The imported file is not a Dart DevTools file.'
     ' At this time, DevTools only supports importing files that were originally'
     ' exported from DevTools.';
 
-String unsupportedDevToolsFileMessage(String devToolsScreen) {
-  return 'Could not import file. The imported file is from "$devToolsScreen", '
-      'which is not supported by this version of Dart DevTools. You may need to'
-      ' upgrade your version of Dart DevTools to view this file.';
+String attemptingToImportMessage(String devToolsScreen) {
+  return 'Attempting to import file for screen with id "$devToolsScreen".';
```

Comment:
add a TODO to specifically track the devtools versions with the snapshots. That way we can concretely say that "the screen was created with DevTools 0.3 but you opened with 0.2 which is likely the reason the screen is missing.

Reply:
added a todo for better error handling and included the devtools version in the export code